call_center.py

```diff
@@ -40,49 +41,98 @@
     Price - 22$
     '''
 
+menu_ld = [{'Traditional': 19}, {'Margherita': 22}, {'Pizza Bianca': 21},
+           {'Marinara': 17}, {'Havaiian': 25}, {'Bruschetta': 22}]
+
```

Comment:
You have the same data in three different places: `menu_full`, `menu_ld` and the JSON file.
The data is the same, just its structure is different, it would be good if you just read the data from JSON file once and then apply needed transformations on it.

Reply:
In `menu_full` you also have `ingredients`, I suggest you add one more key/value pair to the JSON dictionary you have in the file.